lib/docgen/class_template.md.erb

```diff
@@ -0,0 +1,81 @@
+<%= kind %> <%= title %>
+========================
```

Comment:
Looking at the generated page, it seems like the name of the class ends up being shown twice?

![image](https://user-images.githubusercontent.com/71433/79920461-29890980-83fe-11ea-94bd-e32050f727d8.png)

Can we add the kind to the page title somehow, or else make this second title smaller?